.github/workflows/test.yml

```diff
@@ -178,38 +178,30 @@ jobs:
       - name: Setup PostgreSQL root user
         run: sudo --user=postgres createuser --superuser root
 
-      - name: Build local debian package
+      - name: Build package
         uses: linz/linz-software-repository@v9
         with:
           release: ${{ steps.distro.outputs.codename }}
           packages: jq
-        env:
-          PG_SUPPORTED_VERSIONS: ${{ matrix.pg }}
 
       - name: Install postgresql-agnostic tableversion package
-        run: sudo dpkg -i build-area/tableversion_*.deb
+        run: sudo dpkg --install build-area/tableversion_*.deb
 
       - name: Show package contents
-        run: dpkg -L tableversion
+        run: dpkg --listfiles tableversion
 
       - name: Run post-install postgresql-agnostic tests
         run: make installcheck-loader-noext
 
       - name: Install postgresql-specific tableversion package
-        run: sudo dpkg -i build-area/postgresql-${{ matrix.pg }}-tableversion_*.deb
+        run: sudo dpkg --install build-area/postgresql-${{ matrix.pg }}-tableversion_*.deb
 
       - name: Test installation
         run: make installcheck
 
       - name: Test loader
         run: make installcheck-loader
 
-      - name: Build packages for all supported versions
```

Comment:
Was this never needed?

Reply:
No, because the `PG_SUPPORTED_VERSIONS` variable doesn't seem to be referenced anywhere anymore, so the two builds were doing the same thing.